game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/CombatValue.java

```diff
@@ -15,8 +15,14 @@
 
   boolean isDefending();
 
+  Collection<TerritoryEffect> getTerritoryEffects();
```

Comment:
I added this api method because `territoryEffects` was being passed around solely to be used in a "simplified" combat value calculator (see `UnitBattleComparator#getUnitPowerForSorting`).  I think I'll be able to replace that completely with the new combat values but I didn't want to include that change in this PR.  So, by adding this `getTerritoryEffects` api, it allows me to remove the need to pass it around manually and then once I rework `UnitBattleComparator#getUnitPowerForSorting`, I'll remove this api completely.